packages/react/cypress/component/auto/form/AutoPasswordInput.cy.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { api } from "../../../support/api.js";
+import { describeForEachAutoAdapter } from "../../../support/auto.js";
+
+describeForEachAutoAdapter("AutoForm", ({ name, adapter: { AutoForm }, wrapper }) => {
+  beforeEach(() => {
+    cy.viewport("macbook-13");
+  });
+
+  const expectUpdateActionSubmissionVariables = (expectedQueryValue?: any) => {
+    cy.intercept({ method: "POST", url: `${api.connection.endpoint}?operation=updateUser` }, (req) => {
+      // eslint-disable-next-line
+      expect(req.body.variables).to.deep.equal(expectedQueryValue);
+
+      // The response content doesn't matter for the tests
+      req.reply({ data: { updateUser: { success: true, errors: null, x: {} } } });
```

Comment:
It would be easier to read if this is not a one-liner, i.e.:
```js
req.reply({
  data: {
    updateUser: {
      // ...
```

Reply:
Updated such that it is a multi-liner